compiler/scenario-service/server/src/main/scala/com/digitalasset/daml/lf/scenario/Context.scala

```diff
@@ -148,7 +148,7 @@ class Context(val contextId: Context.ContextId) {
   }
 
   def allPackages: Map[PackageId, Ast.Package] =
-    extPackages + (homePackageId -> Ast.Package(modules))
+    extPackages + (homePackageId -> Ast.Package(modules, Set.empty[PackageId]))
```

Comment:
I guess this is fine since we simply don’t care about this in the scenario service? If so it might be worth pointing this out in a comment.

Reply:
I'll add a comment.